package.json

```diff
@@ -18,15 +18,16 @@
     "prettier-format": "prettier --write \"**/*.{js,ts,tsx,json,scss,html}\"",
     "start": "(test -f client/build/index.html || yarn build:client) && (test -f ssr/dist/main.js || yarn build:ssr) && (test -d client/build/en-us/_spas || yarn build --spas-only) && nf -j Procfile.start start",
     "start:client": "cd client && cross-env BROWSER=none PORT=3000 react-scripts start",
-    "start:dev-server": "nodemon server --watch server --watch build --watch content --watch kumascript --ext js",
+    "start:dev-server": "nodemon server --watch server --watch build --watch content --watch kumascript --ext js --watch ssr/dist",
     "start:server": "node server",
     "start:static-server": "cross-env ENV_FILE=testing/.env node server/static.js",
     "test": "yarn prettier-check && yarn test:client && yarn test:kumascript && yarn test:build && yarn test:testing",
     "test:build": "jest build",
     "test:client": "cd client && tsc --noEmit && react-scripts test --env=jest-environment-jsdom-sixteen",
     "test:kumascript": "jest kumascript",
     "test:testing": "cd testing && jest",
-    "tool": "node tool/cli.js"
+    "tool": "node tool/cli.js",
+    "watch:ssr": "cd ssr && webpack --config webpack.config.js --mode=production --watch"
```

Comment:
Not sure if you need to provide the full config file path if it's following the default naming